snowfakery_samples/npsp/BDI_npsp.recipe.yml

```diff
@@ -42,9 +50,9 @@
   include: donation     
   fields:
     npsp__Contact1_Firstname__c:
-      fake: first_name
+      fake: FirstName
     npsp__Contact1_Lastname__c:
-      fake: last_name
+      fake: LastName
     npsp__Contact1_Personal_Email__c:
       fake: safe_email
```

Comment:
@jesslopez-sf remind me - is safe_email the right option here?  I know we talked about how this has been updated, but I can't remember where we landed.

Reply:
As of [v 1.10](https://github.com/SFDO-Tooling/Snowfakery/releases/tag/v1.10) the main email handle makes all emails Safe by default.  There are also apparently now  `danger_mail` and `RealisticMaybeRealEmail` for people who want the previous behavior.